tsconfig.json

```
@@ -6,6 +6,8 @@
     "allowJs": true
   },
   "exclude": [
-    "**/node_modules"
+    "**/node_modules",
+    "integration/bun/**"
```

I think we can ignore all integration code here.

```suggestion
    "integration/**"
```